Consul/KV.cs

```diff
@@ -509,7 +508,7 @@ public Task<WriteResult<bool>> Release(KVPair p, CancellationToken ct = default)
         public Task<WriteResult<bool>> Release(KVPair p, WriteOptions q, CancellationToken ct = default)
         {
             p.Validate();
-            var req = _client.Put<object, bool>(string.Format("/v1/kv/{0}", p.Key.TrimStart('/')), null, q);
+            var req = _client.Put<object, bool>(string.Format("/v1/kv/{0}", p.Key.TrimStart('/')), p.Value, q);
```

Comment:
🐛 
```suggestion
            var req = _client.Put<byte[], bool>(string.Format("/v1/kv/{0}", p.Key.TrimStart('/')), p.Value, q);
```

Reply:
Ah, so that's why when I tried my other tests, they weren't working. Thank you.